servers/physics_2d/space_2d_sw.cpp

```diff
@@ -851,7 +850,7 @@ bool Space2DSW::test_body_motion(Body2DSW *p_body, const Transform2D &p_from, co
 			for (int i = 0; i < cbk.amount; i++) {
 				Vector2 a = sr[i * 2 + 0];
 				Vector2 b = sr[i * 2 + 1];
-				recover_motion += (b - a) * 0.4;
+				recover_motion += (b - a) / cbk.amount;
```

Comment:
Also, it's a bit outside of this PR's scope, but you could possibly change the same code in `test_body_ray_separation` for consistency reasons (it uses 0.4 too). Would it make sense to you? I'm not sure if it's exactly the same purpose but it seems so.